applications/processor/aggregator-processor/README.adoc

```diff
@@ -15,102 +15,8 @@ Otherwise(including a deserialization error), the input payload is left as is -
-$$time-between-eviction-runs$$:: $$Time between runs of the idle object evictor thread. When positive, the idle object evictor thread starts, otherwise no idle object eviction is performed.$$ *($$Duration$$, default: `$$<none>$$`)*
-
-=== spring.data.redis.lettuce.pool
-
-$$enabled$$:: $$Whether to enable the pool. Enabled automatically if "commons-pool2" is available. With Jedis, pooling is implicitly enabled in sentinel mode and this setting only applies to single node setup.$$ *($$Boolean$$, default: `$$<none>$$`)*
-$$max-active$$:: $$Maximum number of connections that can be allocated by the pool at a given time. Use a negative value for no limit.$$ *($$Integer$$, default: `$$8$$`)*
-$$max-idle$$:: $$Maximum number of "idle" connections in the pool. Use a negative value to indicate an unlimited number of idle connections.$$ *($$Integer$$, default: `$$8$$`)*
-$$max-wait$$:: $$Maximum amount of time a connection allocation should block before throwing an exception when the pool is exhausted. Use a negative value to block indefinitely.$$ *($$Duration$$, default: `$$-1ms$$`)*
-$$min-idle$$:: $$Target for the minimum number of idle connections to maintain in the pool. This setting only has an effect if both it and time between eviction runs are positive.$$ *($$Integer$$, default: `$$0$$`)*
-$$time-between-eviction-runs$$:: $$Time between runs of the idle object evictor thread. When positive, the idle object evictor thread starts, otherwise no idle object eviction is performed.$$ *($$Duration$$, default: `$$<none>$$`)*
-
-=== spring.data.redis.lettuce
-
-$$shutdown-timeout$$:: $$Shutdown timeout.$$ *($$Duration$$, default: `$$100ms$$`)*
-
-=== spring.data.redis.sentinel
-
-$$master$$:: $$Name of the Redis server.$$ *($$String$$, default: `$$<none>$$`)*
-$$nodes$$:: $$Comma-separated list of "host:port" pairs.$$ *($$List<String>$$, default: `$$<none>$$`)*
-$$password$$:: $$Password for authenticating with sentinel(s).$$ *($$String$$, default: `$$<none>$$`)*
-$$username$$:: $$Login username for authenticating with sentinel(s).$$ *($$String$$, default: `$$<none>$$`)*
-
-=== spring.data.redis.ssl
-
-$$bundle$$:: $$SSL bundle name.$$ *($$String$$, default: `$$<none>$$`)*
-$$enabled$$:: $$Whether to enable SSL support. Enabled automatically if "bundle" is provided unless specified otherwise.$$ *($$Boolean$$, default: `$$<none>$$`)*
-
-=== spring.datasource
-
-$$driver-class-name$$:: $$Fully qualified name of the JDBC driver. Auto-detected based on the URL by default.$$ *($$String$$, default: `$$<none>$$`)*
-$$password$$:: $$Login password of the database.$$ *($$String$$, default: `$$<none>$$`)*
-$$url$$:: $$JDBC URL of the database.$$ *($$String$$, default: `$$<none>$$`)*
-$$username$$:: $$Login username of the database.$$ *($$String$$, default: `$$<none>$$`)*
+//tag::configuration-properties[link-to-catalog=true]
```

Comment:
> [!NOTE]
> These links are generated w/ an additive change to the apps docs plugin where you set `link-to-catalog=true` and it instead generates a link. I am waiting on feedback to this PR before I push that up for a PR as it will change if there are changes requested here.